src/Language/Haskell/Brittany/Internal.hs

```diff
@@ -405,7 +397,7 @@ parsePrintModuleTests conf filename input = do
         then return $ pPrintModule moduleConf perItemConf anns parsedModule
         else lift
           $ pPrintModuleAndCheck moduleConf perItemConf anns parsedModule
-      if null errs
+      if null $ filter (not . isErrorUnusedComment) errs
```

Comment:
These unused comment errors were only showing up in tests for me. If I copy-pasted the test case into a source file and ran the `brittany` executable against it, I got no warnings or errors.